ensime-client.el

```diff
@@ -1115,6 +1115,10 @@ copies. All other objects are used unchanged. List must not contain cycles."
   (ensime-eval `(swank:shutdown-server)))
 
 (defun ensime-rpc-symbol-designations (file start end requested-types continue)
+  (while (not (ensime-protocol-version))
+    ;; workaround startup timing issue
+    (message "Waiting for connection-info")
+    (sit-for 1))
```

Comment:
That I don't know. I assume the format changed at some point. Happy birthday BTW :smile: I don't know how but my phone seems to know these things